bin/db_patcher.js

```diff
@@ -23,8 +23,8 @@ options.reversePatchAllowed = false
 
 patcher.patch(options, function(err) {
   if (err) {
-    log.error(err)
+    logger.error('patch-error', { err : '' + err })
     process.exit(2)
   }
-  log.info('Database patched to level ' + options.patchLevel)
+  logger.info('Database patched to level ' + options.patchLevel)
```

Comment:
Technically, this violates the heka format: this will set the logger to `bin.db_patcher.Database patched to level 5`, and the spaces are bad.

For this reason, I suggest passing `debug: true` to `mozlog` when running the tests, which will `assert.equal(logger.indexOf(' '), -1)`.